cheroot/testing.py

```diff
@@ -61,18 +63,22 @@ def cheroot_server(server_factory):
     httpserver.stop()  # destroy it
 
 
-@pytest.fixture(scope='module')
-def wsgi_server():
-    """Set up and tear down a Cheroot WSGI server instance."""
-    for srv in cheroot_server(cheroot.wsgi.Server):
-        yield srv
-
-
-@pytest.fixture(scope='module')
-def native_server():
-    """Set up and tear down a Cheroot HTTP server instance."""
-    for srv in cheroot_server(cheroot.server.HTTPServer):
-        yield srv
+def check_log_messages(ignored_msgs, caplog):
```

Comment:
As I wrote earlier, it'd like this module to only contain things that have the potential to be exposed as pytest-fixtures for end-users to test their Cheroot-based apps. I'm not sure if this helper should be here. What do you think?